examples/adv_diff/ex8/write_ascii.cpp

```diff
@@ -0,0 +1,38 @@
+#include <mpi.h>
+
+#include <cassert>
+#include <fstream>
+#include <iostream>
+#include <vector>
+int
+main(int argc, char** argv)
```

Comment:
Could you add a comment block before `main` explaining what this file does? I'm lost in all the MPI file commands.

Reply:
This file is used to read the MPI binary file written by ```example.cpp``` and output it in ASCII file format for postprocessing. I am not good at MPI. If there is a way for me to write the data in ASCII format directly in ```example.cpp```, I am ready to do that. I just followed the code from ```tests/IIM/taylor_couette_2d.cpp``` which writes velocity profile:

https://github.com/IBAMR/IBAMR/blob/d5981eb355272b92c655006616d4c9edaa5a2cae/tests/IIM/taylor_couette_2d.cpp#L991


I used this code snippet for the temperature profile.